Bottom: 71d5ecbf50720877e5e802cbc00c57e840479039
Top:    0f09dd09a8e2073b7edd7aff646f146b36e241f2
Author: Peter Clifton <pcjc2@cam.ac.uk>
Date:   2011-03-27 02:16:50 +0100

crosshair.c: Nasty thing FIXME


---

diff --git a/src/crosshair.c b/src/crosshair.c
index f733e11..6b55cbe 100644
--- a/src/crosshair.c
+++ b/src/crosshair.c
@@ -693,8 +693,10 @@ CrosshairOn (void)
   if (!Crosshair.On)
     {
       Crosshair.On = true;
+#if 0
       DrawAttached ();
       DrawMark ();
+#endif
     }
 }
 
@@ -707,8 +709,10 @@ CrosshairOff (void)
   if (Crosshair.On)
     {
       Crosshair.On = false;
+#if 0
       DrawAttached ();
       DrawMark ();
+#endif
     }
 }
